torch/onnx/utils.py

```diff
@@ -108,36 +128,66 @@ def setup_onnx_logging(verbose):
         if not is_originally_enabled:
             torch.onnx.disable_log()
 
+
 @contextlib.contextmanager
 def exporter_context(model, mode, verbose):
-    with select_model_mode_for_export(model, mode) as mode_ctx, \
-            disable_apex_o2_state_dict_hook(model) as apex_ctx, \
-            setup_onnx_logging(verbose) as log_ctx:
+    with select_model_mode_for_export(
```

Comment:
is this auto-formatted? feels less readable.

Reply:
It is. Black has some discussion on using slashes: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#labels-why-no-backslashes

I agree that this is a case where it feels less readable. But overall, we are trading fine control on how the code looks for freedom from having to worry about formatting in return.